include/eve/module/proba/regular/impl/lognormal_distribution.hpp

```diff
@@ -270,7 +270,7 @@ namespace eve
         return median(d)*exp(-sqr(d.s));
       else
       {
-        value_type inve = I(0.36787944117144232159552377016146086744581113103176);
+        auto inve = I(0.36787944117144232159552377016146086744581113103176);
```

Comment:
this value_type is a local typedef, I think it should stay the way it was.

Reply:
It's not a local typedef, it's likely dead/untested code.

There is no `value_type` defined. It was just never compiled.
And now there's no `value_type` struct, so even in a not-instantiated form it's not working.